javascript/javascript3/week1/Gif/scripts.js

```diff
@@ -0,0 +1,23 @@
+const APIKEY = "HE28dRYots7IvgGuuRC1NR8AEYbHqqyK";
+const searchInput = document.getElementById("search-input");
+const numberInput = document.getElementById("number-input");
+const button = document.getElementById("button");
+const results = document.getElementById("results");
+
+button.addEventListener("click", (btn)=>{
+    btn.preventDefault();
+    fetch(`https://api.giphy.com/v1/gifs/search?api_key=${APIKEY}&limit=${numberInput.value}&lang=en&q=${searchInput.value}`)
```

Comment:
I'd recommend grabbing the input values and storing them as variables first.
It makes the URL easier to read and also gives you a chance to validate those values before putting them into the URL.